<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager">
    <remap from="rgb/camera_info" to="/head_camera/depth/camera_info"/>
    <remap from="rgb/image_rect_color" to="/filtered_image"/>
    <remap from="depth_registered/image_rect" to="/filtered_depth_image"/>
    <remap from="depth_registered/points" to="combined_filtered_image_and_depthmap"/>
  </node>
</launch>
